site-modules/gplt_runner_setup/plans/setup_user.pp

```diff
@@ -52,6 +52,6 @@
 
   #do not try to use _run_as => $username...  For some reason it claims it works but doesn't.
   #the tasks chown and chgrp anyway
-  run_task("gplt_runner_setup::setup_metrics_viewer", $nodes, instance_username => $username)
```

Comment:
Why is the viewer no longer needed? Please comment in the commit message.
Since the viewer is no longer used, I presume that https://github.com/RandellP/gplt_runner_setup/blob/master/site-modules/gplt_runner_setup/tasks/setup_metrics_viewer.sh should be deleted as part of this PR, n'est pas?

Reply:
updated the messaging on why.
Left it in as the viewer would still be used if we were to test the whole ecosystem.  The dependencies are a bit stretchy...  but the collector now is home to the script to import the metrics into influx db.   the dashboard houses the dashboard definitions, and references the import script from the collector.  The viewer does what the dashboard does, but in a container...  and it pulls the dashboard definitions from the dashboard, and references the import script from the collector.  It's all connected. :)

Reply:
![](https://media.giphy.com/media/iAYupOdWXQy5a4nVGk/200w_d.gif)